recipe/meta.yaml

```diff
@@ -23,45 +22,57 @@ build:
   ignore_run_exports:
     - icu
     - expat
+    - jpeg  # [win]
 
 requirements:
   build:
     - {{ compiler('c') }}
-    - automake
-    - libtool
-    - gettext
+    - automake      # [not win]
+    - libtool       # [not win]
+    - gettext       # [not win]
     - pkg-config
-    - make
-    - patch     # [unix]
-    - m2-patch  # [win]
+    - make          # [not win]
+    - patch         # [unix]
+    - m2-patch      # [win]
+    - cmake         # [win]
+    - ninja-base    # [win]
   host:
     - expat
-    - fontconfig
+    - fontconfig    # [not win]
     - freetype
     - icu
     - jpeg
-    - libiconv  # [osx]
+    - libiconv      # [osx]
     - libpng
     - libtiff
-    - libwebp-base
+    - libwebp-base  # [not win]
+    - zlib          # [win]
```

Comment:
Why does it use zlib only on Windows?

Reply:
It was only used on windows because the gd image format was enabled. I disabled it and removed the zlib dependency as this image format is obsolete. (That's why it's off by default no unix builds.)